src/stream_handler/handlers.py

```
@@ -0,0 +1,90 @@
+# Optional: Create a combined handler that can handle both tags and summaries
+class CombinedStreamHandler(StreamHandler):
+    def __init__(self, config: StreamConfig, strategy_type: StreamStrategyType = StreamStrategyType.STRING, **kwargs):
+        super().__init__(config, strategy_type)
+        self.result = {
+            'response_text': ''
+        }
+
+    async def handle_stream(self, response, **kwargs):
+        model = kwargs['model']
+        output_text = ""
+        response_text = ""
```

Ruff (0.8.2)

Local variable `response_text` is assigned to but never used

Remove assignment to unused variable `response_text`

(F841)